big_bang.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -e 
+
+pkg_name=$1
+echo "
+
+ ▄▄▄▄    ██▓  ▄████     ▄▄▄▄    ▄▄▄       ███▄    █   ▄████ 
```

Comment:
LOL. Have you seen this tool? https://patorjk.com/software/taag/#p=display&f=Graffiti&t=Type%20Something%20